### PR TITLE
Updated OIDC Client Details For Audit Purpose

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

File: partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/controller/ClientManagementController.java [hunk 1]
```diff
@@ -22,29 +26,49 @@ public class ClientManagementController {
...
+	@Autowired
+	AuditUtil auditUtil;
+
+	String msg = "mosip.clientmanagement.clients.retrieve";
+
+	String version = "1.0";
+
 	@RequestMapping(value = "/oidc/client", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseWrapper<ClientDetailResponse> createClient(
+	public ResponseEntity<ResponseWrapper<ClientDetailResponse>> createClient(
```
Comment (balaji-alluru) [inline partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/controller/ClientManagementController.java:37]: why you modified the return type?

File: partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/contant/ClientServiceAuditEnum.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
+package io.mosip.pms.oidc.client.contant;
+
+import io.mosip.pms.partner.manager.constant.AuditConstant;
+
+public enum ClientServiceAuditEnum {
+
+    CREATE_CLIENT("PMS_PRT_100", AuditConstant.AUDIT_SYSTEM, "POST CREATE OIDC CLIENT",
```
Comment (balaji-alluru) [inline partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/contant/ClientServiceAuditEnum.java:7]: please check and try to give some consecutive numbers as event id

File: partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/controller/ClientManagementController.java (hunk 1 above)
Comment (GOKULRAJ136) [reply partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/controller/ClientManagementController.java:37]: I need to send a response back to the client after processing a request that's why

File: partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/contant/ClientServiceAuditEnum.java (hunk 2 above)
Comment (GOKULRAJ136) [reply partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/contant/ClientServiceAuditEnum.java]: Updated the Event ID's with the consecutive numbers

File: partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/controller/ClientManagementController.java (hunk 1 above)
Comment (GOKULRAJ136) [reply partner/partner-management-service/src/main/java/io/mosip/pms/oidc/client/controller/ClientManagementController.java]: Changed the return type

**Review by LoganathanSekar7627 — APPROVED** [review]

**Pull request closed without merging**